index.html

```diff
@@ -1,232 +1,395 @@
+          </div>
+        </div>
+        <div class="skill-section-items">
+          <ul>
+            <li>
+              <div class="languages-dropdown-bar">
+                <p>
+                  Langauges<span
+                    ><img src="./images/dropdown..png" alt="languages"
+                  /></span>
+                </p>
+                <div class="skill-items">
+                  <ul>
+                    <li>
+                      <span
+                        ><img
+                          src="./images/JS.png"
+                          alt="js-icon"
+                        />Javascript</span
+                      >
+                    </li>
+                    <li>
+                      <span
+                        ><img
+                          src="./images/HTML.png"
+                          alt="html-icon"
+                        />HTML</span
+                      >
+                    </li>
+                    <li>
+                      <span
+                        ><img src="./images/CSS.png" alt="css-icon" />CSS</span
+                      >
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="skill-projects-item-one">
+                <p>
+                  Frameworks<span
+                    ><img src="./images/arrow.png" alt="frames-icon"
+                  /></span>
+                </p>
+              </div>
+            </li>
+            <li>
+              <div class="skill-projects-item-two">
+                <p>
+                  Skills<span
+                    ><img src="./images/arrow.png" alt="skills-icon"
+                  /></span>
+                </p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </section>
+    <section class="form">
+      <form action="/my-form-page" method="post">
+        <div class="header">
+          <h4>Contact Me</h4>
+          <p class="info">
+            If you have an application you<br />are interested in developing,
+            a,<br />that you need built or a<br />project that needs coding.
+            I’d<br />love to help with it
+          </p>
+        </div>
+        <div class="forms">
+          <ul>
+            <li>
+              <label for="name"></label>
+              <input type="text" id="name" name="user_name" maxlength="30" placeholder="your name " required/>
+            </li>
+            <li>
+              <label for="mail"></label>
+              <input type="email" id="mail" name="user_mail" placeholder="your email " required/>
+            </li>
+            <li>
+              <label for="msg"></label>
+              <textarea class="text" id="msg" name="user_message" maxlength="500" placeholder="write something here" required></textarea>
+            </li>
+            <li>
+              <button class="but" type="submit">Get in Touch</button>
```

Comment:
**[REQUIRED CHANGED]**


- Kindly add the interactions on **ALL** your buttons to indicate the hover, active, enabled, and disabled state to improve the  user experience and to meet the project requirement below:


![buton](https://user-images.githubusercontent.com/59397159/172722065-1a2f00fe-0099-4898-a4dd-d7d57066765d.png)